src/utils/httpRequestParser.ts

```diff
@@ -103,6 +103,17 @@ export class HttpRequestParser implements IRequestParser {
             requestLine.url = `${scheme}://${host}${requestLine.url}`;
         }
 
+        // shorthand syntax for localhost
+        if (requestLine.url[0] === ':') {
+            if (requestLine.url[1] === '/') {
+                requestLine.url = `http://localhost:80${requestLine.url.slice(1)}`;
+            } else {

```

Comment:
How about the case for `:` only?

Reply:
This works too! See the check in line 108: “if `:` is directly followed by `/` (i. e. the path part), treat it as `localhost:80`”.

Reply:
No, Imean the value of `requestLine.url` is `:`, in this case, the final value of it is `http://localhost:`

Reply:
Ah, makes sense! I'll add the changes later today. :sparkles: